google/datalab/contrib/mlworkbench/commands/_ml.py

```diff
@@ -415,6 +417,26 @@ def ml(line, cell=None):
                                    help='Prediction Data. Can be a csv line, or a dict.')
   explain_parser.set_defaults(func=_explain)
 
+  probe_parser = parser.subcommand(
+      'probe',
```

Comment:
The name "probe" for this is not immediately obvious to me.  Why did you choose it?